Installation issue: pango: build cannot detect glib without libffi

### Steps to reproduce the issue

```console
$ spack install pango@1.41.0+X
==> Executing phase: 'autoreconf'
==> Executing phase: 'configure'
==> [2019-08-10-18:00:46.097702] '/Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/configure' '--prefix=/Users/goxberry/repos/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.1-apple/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo' '--without-xft' '--disable-gtk-doc-html' 'GTKDOC_CHECK=/usr/bin/true' 'GTKDOC_CHECK_PATH=/usr/bin/true' 'GTKDOC_MKPDF=/usr/bin/true' 'GTKDOC_REBASE=/usr/bin/true'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/install-sh -c -d
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking build system type... x86_64-apple-darwin18.7.0
checking host system type... x86_64-apple-darwin18.7.0
checking for native Win32... no
checking for gcc... /Users/goxberry/repos/spack/lib/spack/env/clang/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang accepts -g... yes
checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang option to accept ISO C89... none needed
checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of /Users/goxberry/repos/spack/lib/spack/env/clang/clang... gcc3
checking whether we are using the GNU C++ compiler... yes
checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ accepts -g... yes
checking dependency style of /Users/goxberry/repos/spack/lib/spack/env/clang/clang++... gcc3
checking for sysconf... yes
checking for getpagesize... yes
checking how to run the C preprocessor... /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E
checking for grep that handles long lines and -e... /usr/local/bin/ggrep
checking for egrep... /usr/local/bin/ggrep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for unistd.h... (cached) yes
checking sys/mman.h usability... yes
checking sys/mman.h presence... yes
checking for sys/mman.h... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/local/bin/gsed
checking for fgrep... /usr/local/bin/ggrep -F
checking for ld used by /Users/goxberry/repos/spack/lib/spack/env/clang/clang... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 196608
checking how to convert x86_64-apple-darwin18.7.0 file names to x86_64-apple-darwin18.7.0 format... func_convert_file_noop
checking how to convert x86_64-apple-darwin18.7.0 file names to toolchain format... func_convert_file_noop
checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... dlltool
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... no
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from /Users/goxberry/repos/spack/lib/spack/env/clang/clang object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for lipo... lipo
checking for otool... otool
checking for otool64... no
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking for -force_load linker flag... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -fno-rtti -fno-exceptions... yes
checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang option to produce PIC... -fno-common -DPIC
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang PIC flag -fno-common -DPIC works... yes
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang static flag -static works... no
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -c -o file.o... yes
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -c -o file.o... (cached) yes
checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin18.7.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E
checking for ld used by /Users/goxberry/repos/spack/lib/spack/env/clang/clang++... /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld... no
checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ option to produce PIC... -fno-common -DPIC
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ PIC flag -fno-common -DPIC works... yes
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ static flag -static works... no
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ supports -c -o file.o... yes
checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ supports -c -o file.o... (cached) yes
checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin18.7.0 dyld
checking how to hardcode library paths into programs... immediate
configure: creating ./config.lt
config.lt: creating libtool
checking for some Win32 platform... no
checking for pkg-config... /usr/local/opt/pkg-config/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for HARFBUZZ... yes
checking for FONTCONFIG... yes
checking for FcWeightFromOpenTypeDouble... yes
checking for FREETYPE... yes
checking for XFT... no
checking for CoreText availability... yes
checking for CAIRO... no
checking for GLIB... no
configure: error: 
*** Glib 2.33.12 or better is required. The latest version of
*** Glib is always available from ftp://ftp.gtk.org/.
$ cat $(spack location -s pango@1.41.0)/spack-src/config.log
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by pango configure 1.41.0, which was
generated by GNU Autoconf 2.69.  Invocation command line was

  $ /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/configure --prefix=/Users/goxberry/repos/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.1-apple/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo --without-xft --disable-gtk-doc-html GTKDOC_CHECK=/usr/bin/true GTKDOC_CHECK_PATH=/usr/bin/true GTKDOC_MKPDF=/usr/bin/true GTKDOC_REBASE=/usr/bin/true

## --------- ##
## Platform. ##
## --------- ##

hostname = Geoffs-MBP.hsd1.ca.comcast.net
uname -m = x86_64
uname -r = 18.7.0
uname -s = Darwin
uname -v = Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64

/usr/bin/uname -p = i386
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = Mach kernel version:
	 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64
Kernel configured for up to 12 processors.
6 processors are physically available.
12 processors are logically available.
Processor type: x86_64h (Intel x86-64h Haswell)
Processors active: 0 1 2 3 4 5 6 7 8 9 10 11
Primary memory available: 32.00 gigabytes
Default processor set: 542 tasks, 2784 threads, 12 processors
Load average: 3.58, Mach factor: 8.40
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /Users/goxberry/repos/spack/lib/spack/env/clang
PATH: /Users/goxberry/repos/spack/lib/spack/env/case-insensitive
PATH: /Users/goxberry/repos/spack/lib/spack/env
PATH: /usr/local/opt/pkg-config/bin
PATH: /usr/local/opt/gobject-introspection/bin
PATH: /usr/local/opt/glib/bin
PATH: /usr/local/opt/harfbuzz/bin
PATH: /usr/local/opt/cairo/bin
PATH: /usr/local/opt/pkg-config/bin
PATH: /usr/local/opt/gobject-introspection/bin
PATH: /usr/local/opt/glib/bin
PATH: /usr/local/opt/harfbuzz/bin
PATH: /usr/local/opt/cairo/bin
PATH: /Users/goxberry/.goenv/shims
PATH: /Users/goxberry/.goenv/bin
PATH: /Users/goxberry/.nodenv/shims
PATH: /Users/goxberry/.nodenv/bin
PATH: /Users/goxberry/.rbenv/shims
PATH: /Users/goxberry/.rbenv/bin
PATH: /Users/goxberry/.pyenv/shims
PATH: /Users/goxberry/.pyenv/bin
PATH: /Users/goxberry/.jenv/shims
PATH: /Users/goxberry/.jenv/bin
PATH: /Users/goxberry/repos/spack
PATH: /Users/goxberry/Library/Python/3.7/bin
PATH: /Users/goxberry/Library/Python/2.7/bin
PATH: /Users/goxberry/.npm-packages/bin
PATH: /Users/goxberry/.gem/ruby/2.3.0/bin
PATH: /usr/local/bin
PATH: /usr/local/sbin
PATH: /usr/local/bin
PATH: /usr/local/bin
PATH: /usr/bin
PATH: /bin
PATH: /usr/sbin
PATH: /sbin
PATH: /Library/TeX/texbin
PATH: /usr/local/share/dotnet
PATH: /opt/X11/bin
PATH: ~/.dotnet/tools
PATH: /Library/Frameworks/Mono.framework/Versions/Current/Commands
PATH: /Users/goxberry/.goenv/shims
PATH: /Users/goxberry/.goenv/bin
PATH: /Users/goxberry/.nodenv/shims
PATH: /Users/goxberry/.nodenv/bin
PATH: /Users/goxberry/.rbenv/shims
PATH: /Users/goxberry/.rbenv/bin
PATH: /Users/goxberry/.pyenv/shims
PATH: /Users/goxberry/.pyenv/bin
PATH: /Users/goxberry/.jenv/shims
PATH: /Users/goxberry/.jenv/bin
PATH: /Users/goxberry/repos/spack/bin
PATH: /Users/goxberry/repos/spack
PATH: /Users/goxberry/Library/Python/3.7/bin
PATH: /Users/goxberry/Library/Python/2.7/bin
PATH: /Users/goxberry/.npm-packages/bin
PATH: /Users/goxberry/.gem/ruby/2.3.0/bin
PATH: /usr/local/sbin
PATH: /Library/TeX/texbin/
PATH: /opt/pgi/osx86-64/19.4/bin
PATH: /opt/X11/bin
PATH: /Library/TeX/texbin/
PATH: /opt/pgi/osx86-64/19.4/bin


## ----------- ##
## Core tests. ##
## ----------- ##

configure:2585: checking for a BSD-compatible install
configure:2653: result: /usr/bin/install -c
configure:2664: checking whether build environment is sane
configure:2719: result: yes
configure:2870: checking for a thread-safe mkdir -p
configure:2909: result: /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/install-sh -c -d
configure:2916: checking for gawk
configure:2932: found /usr/local/bin/gawk
configure:2943: result: gawk
configure:2954: checking whether make sets $(MAKE)
configure:2976: result: yes
configure:3005: checking whether make supports nested variables
configure:3022: result: yes
configure:3159: checking whether make supports nested variables
configure:3176: result: yes
configure:3236: checking build system type
configure:3250: result: x86_64-apple-darwin18.7.0
configure:3270: checking host system type
configure:3283: result: x86_64-apple-darwin18.7.0
configure:3304: checking for native Win32
configure:3322: result: no
configure:3384: checking for gcc
configure:3411: result: /Users/goxberry/repos/spack/lib/spack/env/clang/clang
configure:3640: checking for C compiler version
configure:3649: /Users/goxberry/repos/spack/lib/spack/env/clang/clang --version >&5
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:3660: $? = 0
configure:3649: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -v >&5
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:3660: $? = 0
configure:3649: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -V >&5
clang: error: unsupported option '-V -Wno-atomic-implicit-seq-cst'
clang: error: no input files
configure:3660: $? = 1
configure:3649: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -qversion >&5
clang: error: unknown argument '-qversion', did you mean '--version'?
configure:3660: $? = 1
configure:3680: checking whether the C compiler works
configure:3702: /Users/goxberry/repos/spack/lib/spack/env/clang/clang    conftest.c  >&5
configure:3706: $? = 0
configure:3754: result: yes
configure:3757: checking for C compiler default output file name
configure:3759: result: a.out
configure:3765: checking for suffix of executables
configure:3772: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest    conftest.c  >&5
configure:3776: $? = 0
configure:3798: result: 
configure:3820: checking whether we are cross compiling
configure:3828: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest    conftest.c  >&5
configure:3832: $? = 0
configure:3839: ./conftest
configure:3843: $? = 0
configure:3858: result: no
configure:3863: checking for suffix of object files
configure:3885: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c   conftest.c >&5
configure:3889: $? = 0
configure:3910: result: o
configure:3914: checking whether we are using the GNU C compiler
configure:3933: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c   conftest.c >&5
configure:3933: $? = 0
configure:3942: result: yes
configure:3951: checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang accepts -g
configure:3971: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g  conftest.c >&5
configure:3971: $? = 0
configure:4012: result: yes
configure:4029: checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang option to accept ISO C89
configure:4092: /Users/goxberry/repos/spack/lib/spack/env/clang/clang  -c -g -O2  conftest.c >&5
configure:4092: $? = 0
configure:4105: result: none needed
configure:4130: checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang understands -c and -o together
configure:4152: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c conftest.c -o conftest2.o
configure:4155: $? = 0
configure:4152: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c conftest.c -o conftest2.o
configure:4155: $? = 0
configure:4167: result: yes
configure:4195: checking for style of include used by make
configure:4223: result: GNU
configure:4249: checking dependency style of /Users/goxberry/repos/spack/lib/spack/env/clang/clang
configure:4360: result: gcc3
configure:4595: checking for C++ compiler version
configure:4604: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ --version >&5
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:4615: $? = 0
configure:4604: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -v >&5
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:4615: $? = 0
configure:4604: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -V >&5
clang: error: unsupported option '-V -Wno-atomic-implicit-seq-cst'
clang: error: no input files
configure:4615: $? = 1
configure:4604: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -qversion >&5
clang: error: unknown argument '-qversion', did you mean '--version'?
configure:4615: $? = 1
configure:4619: checking whether we are using the GNU C++ compiler
configure:4638: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c   conftest.cpp >&5
configure:4638: $? = 0
configure:4647: result: yes
configure:4656: checking whether /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ accepts -g
configure:4676: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c -g  conftest.cpp >&5
configure:4676: $? = 0
configure:4717: result: yes
configure:4742: checking dependency style of /Users/goxberry/repos/spack/lib/spack/env/clang/clang++
configure:4853: result: gcc3
configure:4880: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c -g -O2  conftest.cpp >&5
configure:4880: $? = 0
configure:4908: checking for sysconf
configure:4908: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2   conftest.c  >&5
configure:4908: $? = 0
configure:4908: result: yes
configure:4908: checking for getpagesize
configure:4908: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2   conftest.c  >&5
configure:4908: $? = 0
configure:4908: result: yes
configure:4922: checking how to run the C preprocessor
configure:4953: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E  conftest.c
configure:4953: $? = 0
configure:4967: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E  conftest.c
conftest.c:18:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:4967: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "pango"
| #define PACKAGE_TARNAME "pango"
| #define PACKAGE_VERSION "1.41.0"
| #define PACKAGE_STRING "pango 1.41.0"
| #define PACKAGE_BUGREPORT "http://bugzilla.gnome.org/enter_bug.cgi?product=pango"
| #define PACKAGE_URL ""
| #define PACKAGE "pango"
| #define VERSION "1.41.0"
| #define PANGO_BINARY_AGE 4100
| #define PANGO_INTERFACE_AGE 0
| #define PANGO_VERSION_MAJOR 1
| #define PANGO_VERSION_MINOR 41
| #define PANGO_VERSION_MICRO 0
| #define HAVE_SYSCONF 1
| #define HAVE_GETPAGESIZE 1
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:4992: result: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E
configure:5012: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E  conftest.c
configure:5012: $? = 0
configure:5026: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E  conftest.c
conftest.c:18:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:5026: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "pango"
| #define PACKAGE_TARNAME "pango"
| #define PACKAGE_VERSION "1.41.0"
| #define PACKAGE_STRING "pango 1.41.0"
| #define PACKAGE_BUGREPORT "http://bugzilla.gnome.org/enter_bug.cgi?product=pango"
| #define PACKAGE_URL ""
| #define PACKAGE "pango"
| #define VERSION "1.41.0"
| #define PANGO_BINARY_AGE 4100
| #define PANGO_INTERFACE_AGE 0
| #define PANGO_VERSION_MAJOR 1
| #define PANGO_VERSION_MINOR 41
| #define PANGO_VERSION_MICRO 0
| #define HAVE_SYSCONF 1
| #define HAVE_GETPAGESIZE 1
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:5055: checking for grep that handles long lines and -e
configure:5113: result: /usr/local/bin/ggrep
configure:5118: checking for egrep
configure:5180: result: /usr/local/bin/ggrep -E
configure:5185: checking for ANSI C header files
configure:5205: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5205: $? = 0
configure:5278: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2   conftest.c  >&5
configure:5278: $? = 0
configure:5278: ./conftest
configure:5278: $? = 0
configure:5289: result: yes
configure:5302: checking for sys/types.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for sys/stat.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for stdlib.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for string.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for memory.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for strings.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for inttypes.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for stdint.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5302: checking for unistd.h
configure:5302: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5302: $? = 0
configure:5302: result: yes
configure:5317: checking for unistd.h
configure:5317: result: yes
configure:5317: checking sys/mman.h usability
configure:5317: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:5317: $? = 0
configure:5317: result: yes
configure:5317: checking sys/mman.h presence
configure:5317: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -E  conftest.c
configure:5317: $? = 0
configure:5317: result: yes
configure:5317: checking for sys/mman.h
configure:5317: result: yes
configure:5714: checking how to print strings
configure:5741: result: printf
configure:5762: checking for a sed that does not truncate output
configure:5826: result: /usr/local/bin/gsed
configure:5844: checking for fgrep
configure:5906: result: /usr/local/bin/ggrep -F
configure:5941: checking for ld used by /Users/goxberry/repos/spack/lib/spack/env/clang/clang
configure:6008: result: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
configure:6015: checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld
configure:6030: result: no
configure:6042: checking for BSD- or MS-compatible name lister (nm)
configure:6096: result: /usr/bin/nm -B
configure:6226: checking the name lister (/usr/bin/nm -B) interface
configure:6233: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:6236: /usr/bin/nm -B "conftest.o"
configure:6239: output
0000000000000218 S _some_variable
configure:6246: result: BSD nm
configure:6249: checking whether ln -s works
configure:6253: result: yes
configure:6261: checking the maximum length of command line arguments
configure:6392: result: 196608
configure:6440: checking how to convert x86_64-apple-darwin18.7.0 file names to x86_64-apple-darwin18.7.0 format
configure:6480: result: func_convert_file_noop
configure:6487: checking how to convert x86_64-apple-darwin18.7.0 file names to toolchain format
configure:6507: result: func_convert_file_noop
configure:6514: checking for /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld option to reload object files
configure:6521: result: -r
configure:6595: checking for objdump
configure:6622: result: objdump
configure:6651: checking how to recognize dependent libraries
configure:6851: result: pass_all
configure:6936: checking for dlltool
configure:6963: result: dlltool
configure:6993: checking how to associate runtime and link libraries
configure:7020: result: printf %s\n
configure:7080: checking for ar
configure:7096: found /usr/bin/ar
configure:7107: result: ar
configure:7144: checking for archiver @FILE support
configure:7161: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:7161: $? = 0
configure:7164: ar cru libconftest.a @conftest.lst >&5
ar: @conftest.lst: No such file or directory
configure:7167: $? = 1
configure:7187: result: no
configure:7245: checking for strip
configure:7261: found /usr/bin/strip
configure:7272: result: strip
configure:7344: checking for ranlib
configure:7360: found /usr/bin/ranlib
configure:7371: result: ranlib
configure:7473: checking command to parse /usr/bin/nm -B output from /Users/goxberry/repos/spack/lib/spack/env/clang/clang object
configure:7626: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:7629: $? = 0
configure:7633: /usr/bin/nm -B conftest.o \| sed -n -e 's/^.*[ ]\([BCDEGRST][BCDEGRST]*\)[ ][ ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' | sed '/ __gnu_lto/d' \> conftest.nm
configure:7636: $? = 0
cannot find nm_test_var in conftest.nm
configure:7626: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:7629: $? = 0
configure:7633: /usr/bin/nm -B conftest.o \| sed -n -e 's/^.*[ ]\([BCDEGRST][BCDEGRST]*\)[ ][ ]*_\([_A-Za-z][_A-Za-z0-9]*\)$/\1 _\2 \2/p' | sed '/ __gnu_lto/d' \> conftest.nm
configure:7636: $? = 0
configure:7702: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2   conftest.c conftstm.o >&5
warning: (x86_64)  could not find object file symbol for symbol _main
configure:7705: $? = 0
configure:7743: result: ok
configure:7790: checking for sysroot
configure:7820: result: no
configure:7827: checking for a working dd
configure:7865: result: /bin/dd
configure:7869: checking how to truncate binary pipes
configure:7884: result: /bin/dd bs=4096 count=1
configure:8213: checking for mt
configure:8243: result: no
configure:8263: checking if : is a manifest tool
configure:8269: : '-?'
configure:8277: result: no
configure:8333: checking for dsymutil
configure:8349: found /usr/bin/dsymutil
configure:8360: result: dsymutil
configure:8425: checking for nmedit
configure:8441: found /usr/bin/nmedit
configure:8452: result: nmedit
configure:8517: checking for lipo
configure:8533: found /usr/bin/lipo
configure:8544: result: lipo
configure:8609: checking for otool
configure:8625: found /usr/bin/otool
configure:8636: result: otool
configure:8701: checking for otool64
configure:8731: result: no
configure:8776: checking for -single_module linker flag
/Users/goxberry/repos/spack/lib/spack/env/clang/clang -g -O2  -o libconftest.dylib -dynamiclib -Wl,-single_module conftest.c
configure:8809: result: yes
configure:8812: checking for -exported_symbols_list linker flag
configure:8832: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2   -Wl,-exported_symbols_list,conftest.sym conftest.c  >&5
configure:8832: $? = 0
configure:8842: result: yes
configure:8845: checking for -force_load linker flag
/Users/goxberry/repos/spack/lib/spack/env/clang/clang -g -O2 -c -o conftest.o conftest.c
ar cru libconftest.a conftest.o
ranlib libconftest.a
/Users/goxberry/repos/spack/lib/spack/env/clang/clang -g -O2  -o conftest conftest.c -Wl,-force_load,./libconftest.a
configure:8877: result: yes
configure:8951: checking for dlfcn.h
configure:8951: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  conftest.c >&5
configure:8951: $? = 0
configure:8951: result: yes
configure:9185: checking for objdir
configure:9200: result: .libs
configure:9460: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -fno-rtti -fno-exceptions
configure:9478: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  -fno-rtti -fno-exceptions conftest.c >&5
configure:9482: $? = 0
configure:9495: result: yes
configure:9853: checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang option to produce PIC
configure:9860: result: -fno-common -DPIC
configure:9868: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang PIC flag -fno-common -DPIC works
configure:9886: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  -fno-common -DPIC -DPIC conftest.c >&5
configure:9890: $? = 0
configure:9903: result: yes
configure:9932: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang static flag -static works
configure:9960: result: no
configure:9975: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -c -o file.o
configure:9996: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -c -g -O2  -o out/conftest2.o conftest.c >&5
configure:10000: $? = 0
configure:10022: result: yes
configure:10030: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang supports -c -o file.o
configure:10077: result: yes
configure:10110: checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries
configure:11369: result: yes
configure:11609: checking dynamic linker characteristics
configure:12430: result: darwin18.7.0 dyld
configure:12552: checking how to hardcode library paths into programs
configure:12577: result: immediate
configure:13125: checking whether stripping libraries is possible
configure:13139: result: yes
configure:13165: checking if libtool supports shared libraries
configure:13167: result: yes
configure:13170: checking whether to build shared libraries
configure:13195: result: yes
configure:13198: checking whether to build static libraries
configure:13202: result: no
configure:13225: checking how to run the C++ preprocessor
configure:13252: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E  conftest.cpp
configure:13252: $? = 0
configure:13266: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E  conftest.cpp
conftest.cpp:32:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:13266: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "pango"
| #define PACKAGE_TARNAME "pango"
| #define PACKAGE_VERSION "1.41.0"
| #define PACKAGE_STRING "pango 1.41.0"
| #define PACKAGE_BUGREPORT "http://bugzilla.gnome.org/enter_bug.cgi?product=pango"
| #define PACKAGE_URL ""
| #define PACKAGE "pango"
| #define VERSION "1.41.0"
| #define PANGO_BINARY_AGE 4100
| #define PANGO_INTERFACE_AGE 0
| #define PANGO_VERSION_MAJOR 1
| #define PANGO_VERSION_MINOR 41
| #define PANGO_VERSION_MICRO 0
| #define HAVE_SYSCONF 1
| #define HAVE_GETPAGESIZE 1
| #define STDC_HEADERS 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_MEMORY_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_SYS_MMAN_H 1
| #define HAVE_DLFCN_H 1
| #define LT_OBJDIR ".libs/"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:13291: result: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E
configure:13311: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E  conftest.cpp
configure:13311: $? = 0
configure:13325: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E  conftest.cpp
conftest.cpp:32:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:13325: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "pango"
| #define PACKAGE_TARNAME "pango"
| #define PACKAGE_VERSION "1.41.0"
| #define PACKAGE_STRING "pango 1.41.0"
| #define PACKAGE_BUGREPORT "http://bugzilla.gnome.org/enter_bug.cgi?product=pango"
| #define PACKAGE_URL ""
| #define PACKAGE "pango"
| #define VERSION "1.41.0"
| #define PANGO_BINARY_AGE 4100
| #define PANGO_INTERFACE_AGE 0
| #define PANGO_VERSION_MAJOR 1
| #define PANGO_VERSION_MINOR 41
| #define PANGO_VERSION_MICRO 0
| #define HAVE_SYSCONF 1
| #define HAVE_GETPAGESIZE 1
| #define STDC_HEADERS 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_MEMORY_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_SYS_MMAN_H 1
| #define HAVE_DLFCN_H 1
| #define LT_OBJDIR ".libs/"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:13487: checking for ld used by /Users/goxberry/repos/spack/lib/spack/env/clang/clang++
configure:13554: result: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
configure:13561: checking if the linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) is GNU ld
configure:13576: result: no
configure:13631: checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries
configure:14704: result: yes
configure:14740: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c -g -O2  conftest.cpp >&5
configure:14743: $? = 0
configure:15224: checking for /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ option to produce PIC
configure:15231: result: -fno-common -DPIC
configure:15239: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ PIC flag -fno-common -DPIC works
configure:15257: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c -g -O2  -fno-common -DPIC -DPIC conftest.cpp >&5
configure:15261: $? = 0
configure:15274: result: yes
configure:15297: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ static flag -static works
configure:15325: result: no
configure:15337: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ supports -c -o file.o
configure:15358: /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -c -g -O2  -o out/conftest2.o conftest.cpp >&5
configure:15362: $? = 0
configure:15384: result: yes
configure:15389: checking if /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ supports -c -o file.o
configure:15436: result: yes
configure:15466: checking whether the /Users/goxberry/repos/spack/lib/spack/env/clang/clang++ linker (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld) supports shared libraries
configure:15506: result: yes
configure:15647: checking dynamic linker characteristics
configure:16395: result: darwin18.7.0 dyld
configure:16460: checking how to hardcode library paths into programs
configure:16485: result: immediate
configure:16547: creating ./config.lt

## ------------------ ##
## Running config.lt. ##
## ------------------ ##
config.lt:681: creating libtool
configure:18104: checking for some Win32 platform
configure:18114: result: no
configure:18375: checking for pkg-config
configure:18393: found /usr/local/opt/pkg-config/bin/pkg-config
configure:18405: result: /usr/local/opt/pkg-config/bin/pkg-config
configure:18430: checking pkg-config is at least version 0.9.0
configure:18433: result: yes
configure:18443: checking for HARFBUZZ
configure:18450: $PKG_CONFIG --exists --print-errors "harfbuzz >= 1.2.3"
configure:18453: $? = 0
configure:18467: $PKG_CONFIG --exists --print-errors "harfbuzz >= 1.2.3"
configure:18470: $? = 0
configure:18508: result: yes
configure:18528: checking for FONTCONFIG
configure:18535: $PKG_CONFIG --exists --print-errors "fontconfig >= 2.11.91"
configure:18538: $? = 0
configure:18552: $PKG_CONFIG --exists --print-errors "fontconfig >= 2.11.91"
configure:18555: $? = 0
configure:18593: result: yes
configure:18603: checking for FcWeightFromOpenTypeDouble
configure:18603: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2 -Wall   conftest.c  -L/usr/local/Cellar/fontconfig/2.13.1/lib -L/usr/local/opt/freetype/lib -lfontconfig -lfreetype >&5
configure:18603: $? = 0
configure:18603: result: yes
configure:18624: checking for FREETYPE
configure:18631: $PKG_CONFIG --exists --print-errors "freetype2"
configure:18634: $? = 0
configure:18648: $PKG_CONFIG --exists --print-errors "freetype2"
configure:18651: $? = 0
configure:18689: result: yes
configure:18723: checking for XFT
configure:18730: $PKG_CONFIG --exists --print-errors "xft >= 2.0.0 xrender"
Package xft was not found in the pkg-config search path.
Perhaps you should add the directory containing `xft.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xft' found
Package xrender was not found in the pkg-config search path.
Perhaps you should add the directory containing `xrender.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xrender' found
configure:18733: $? = 1
configure:18747: $PKG_CONFIG --exists --print-errors "xft >= 2.0.0 xrender"
Package xft was not found in the pkg-config search path.
Perhaps you should add the directory containing `xft.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xft' found
Package xrender was not found in the pkg-config search path.
Perhaps you should add the directory containing `xrender.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xrender' found
configure:18750: $? = 1
configure:18764: result: no
No package 'xft' found
No package 'xrender' found
configure:18868: checking for CoreText availability
configure:18883: /Users/goxberry/repos/spack/lib/spack/env/clang/clang -o conftest -g -O2 -Wall   conftest.c  -framework ApplicationServices >&5
configure:18883: $? = 0
configure:18898: result: yes
configure:18935: checking for CAIRO
configure:18942: $PKG_CONFIG --exists --print-errors "cairo >= $cairo_required"
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'gobject-2.0', not found
configure:18945: $? = 1
configure:18959: $PKG_CONFIG --exists --print-errors "cairo >= $cairo_required"
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'gobject-2.0', not found
configure:18962: $? = 1
configure:18976: result: no
Package 'libffi', required by 'gobject-2.0', not found
configure:19256: checking for GLIB
configure:19263: $PKG_CONFIG --exists --print-errors "$GLIB_MODULES"
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'gobject-2.0', not found
configure:19266: $? = 1
configure:19280: $PKG_CONFIG --exists --print-errors "$GLIB_MODULES"
Package libffi was not found in the pkg-config search path.
Perhaps you should add the directory containing `libffi.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libffi', required by 'gobject-2.0', not found
configure:19283: $? = 1
configure:19297: result: no
Package 'libffi', required by 'gobject-2.0', not found
configure:19313: error: 
*** Glib 2.33.12 or better is required. The latest version of
*** Glib is always available from ftp://ftp.gtk.org/.

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_build=x86_64-apple-darwin18.7.0
ac_cv_c_compiler_gnu=yes
ac_cv_cxx_compiler_gnu=yes
ac_cv_env_CAIRO_CFLAGS_set=
ac_cv_env_CAIRO_CFLAGS_value=
ac_cv_env_CAIRO_LIBS_set=
ac_cv_env_CAIRO_LIBS_value=
ac_cv_env_CCC_set=
ac_cv_env_CCC_value=
ac_cv_env_CC_set=set
ac_cv_env_CC_value=/Users/goxberry/repos/spack/lib/spack/env/clang/clang
ac_cv_env_CFLAGS_set=
ac_cv_env_CFLAGS_value=
ac_cv_env_CPPFLAGS_set=
ac_cv_env_CPPFLAGS_value=
ac_cv_env_CPP_set=
ac_cv_env_CPP_value=
ac_cv_env_CXXCPP_set=
ac_cv_env_CXXCPP_value=
ac_cv_env_CXXFLAGS_set=
ac_cv_env_CXXFLAGS_value=
ac_cv_env_CXX_set=set
ac_cv_env_CXX_value=/Users/goxberry/repos/spack/lib/spack/env/clang/clang++
ac_cv_env_FONTCONFIG_CFLAGS_set=
ac_cv_env_FONTCONFIG_CFLAGS_value=
ac_cv_env_FONTCONFIG_LIBS_set=
ac_cv_env_FONTCONFIG_LIBS_value=
ac_cv_env_FREETYPE_CFLAGS_set=
ac_cv_env_FREETYPE_CFLAGS_value=
ac_cv_env_FREETYPE_LIBS_set=
ac_cv_env_FREETYPE_LIBS_value=
ac_cv_env_GLIB_CFLAGS_set=
ac_cv_env_GLIB_CFLAGS_value=
ac_cv_env_GLIB_LIBS_set=
ac_cv_env_GLIB_LIBS_value=
ac_cv_env_GTKDOC_DEPS_CFLAGS_set=
ac_cv_env_GTKDOC_DEPS_CFLAGS_value=
ac_cv_env_GTKDOC_DEPS_LIBS_set=
ac_cv_env_GTKDOC_DEPS_LIBS_value=
ac_cv_env_HARFBUZZ_CFLAGS_set=
ac_cv_env_HARFBUZZ_CFLAGS_value=
ac_cv_env_HARFBUZZ_LIBS_set=
ac_cv_env_HARFBUZZ_LIBS_value=
ac_cv_env_LDFLAGS_set=
ac_cv_env_LDFLAGS_value=
ac_cv_env_LIBS_set=
ac_cv_env_LIBS_value=
ac_cv_env_LIBTHAI_CFLAGS_set=
ac_cv_env_LIBTHAI_CFLAGS_value=
ac_cv_env_LIBTHAI_LIBS_set=
ac_cv_env_LIBTHAI_LIBS_value=
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_PKG_CONFIG_LIBDIR_set=
ac_cv_env_PKG_CONFIG_LIBDIR_value=
ac_cv_env_PKG_CONFIG_PATH_set=set
ac_cv_env_PKG_CONFIG_PATH_value=/usr/local/opt/cairo/lib/pkgconfig:/usr/local/opt/gobject-introspection/lib/pkgconfig:/usr/local/opt/glib/lib/pkgconfig:/usr/local/opt/harfbuzz/lib/pkgconfig
ac_cv_env_PKG_CONFIG_set=
ac_cv_env_PKG_CONFIG_value=
ac_cv_env_XFT_CFLAGS_set=
ac_cv_env_XFT_CFLAGS_value=
ac_cv_env_XFT_LIBS_set=
ac_cv_env_XFT_LIBS_value=
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_func_FcWeightFromOpenTypeDouble=yes
ac_cv_func_getpagesize=yes
ac_cv_func_sysconf=yes
ac_cv_header_dlfcn_h=yes
ac_cv_header_inttypes_h=yes
ac_cv_header_memory_h=yes
ac_cv_header_stdc=yes
ac_cv_header_stdint_h=yes
ac_cv_header_stdlib_h=yes
ac_cv_header_string_h=yes
ac_cv_header_strings_h=yes
ac_cv_header_sys_mman_h=yes
ac_cv_header_sys_stat_h=yes
ac_cv_header_sys_types_h=yes
ac_cv_header_unistd_h=yes
ac_cv_host=x86_64-apple-darwin18.7.0
ac_cv_objext=o
ac_cv_path_EGREP='/usr/local/bin/ggrep -E'
ac_cv_path_FGREP='/usr/local/bin/ggrep -F'
ac_cv_path_GREP=/usr/local/bin/ggrep
ac_cv_path_SED=/usr/local/bin/gsed
ac_cv_path_ac_pt_PKG_CONFIG=/usr/local/opt/pkg-config/bin/pkg-config
ac_cv_path_install='/usr/bin/install -c'
ac_cv_path_lt_DD=/bin/dd
ac_cv_prog_AWK=gawk
ac_cv_prog_CPP='/Users/goxberry/repos/spack/lib/spack/env/clang/clang -E'
ac_cv_prog_CXXCPP='/Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E'
ac_cv_prog_ac_ct_AR=ar
ac_cv_prog_ac_ct_CC=/Users/goxberry/repos/spack/lib/spack/env/clang/clang
ac_cv_prog_ac_ct_DLLTOOL=dlltool
ac_cv_prog_ac_ct_DSYMUTIL=dsymutil
ac_cv_prog_ac_ct_LIPO=lipo
ac_cv_prog_ac_ct_NMEDIT=nmedit
ac_cv_prog_ac_ct_OBJDUMP=objdump
ac_cv_prog_ac_ct_OTOOL=otool
ac_cv_prog_ac_ct_RANLIB=ranlib
ac_cv_prog_ac_ct_STRIP=strip
ac_cv_prog_cc_c89=
ac_cv_prog_cc_g=yes
ac_cv_prog_cxx_g=yes
ac_cv_prog_make_make_set=yes
am_cv_CC_dependencies_compiler_type=gcc3
am_cv_CXX_dependencies_compiler_type=gcc3
am_cv_make_support_nested_variables=yes
am_cv_prog_cc_c_o=yes
lt_cv_apple_cc_single_mod=yes
lt_cv_ar_at_file=no
lt_cv_deplibs_check_method=pass_all
lt_cv_file_magic_cmd='$MAGIC_CMD'
lt_cv_file_magic_test_file=
lt_cv_ld_exported_symbols_list=yes
lt_cv_ld_force_load=yes
lt_cv_ld_reload_flag=-r
lt_cv_nm_interface='BSD nm'
lt_cv_objdir=.libs
lt_cv_path_LD=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
lt_cv_path_LDCXX=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld
lt_cv_path_NM='/usr/bin/nm -B'
lt_cv_path_mainfest_tool=no
lt_cv_prog_compiler_c_o=yes
lt_cv_prog_compiler_c_o_CXX=yes
lt_cv_prog_compiler_pic='-fno-common -DPIC'
lt_cv_prog_compiler_pic_CXX='-fno-common -DPIC'
lt_cv_prog_compiler_pic_works=yes
lt_cv_prog_compiler_pic_works_CXX=yes
lt_cv_prog_compiler_rtti_exceptions=yes
lt_cv_prog_compiler_static_works=no
lt_cv_prog_compiler_static_works_CXX=no
lt_cv_prog_gnu_ld=no
lt_cv_prog_gnu_ldcxx=no
lt_cv_sharedlib_from_linklib_cmd='printf %s\n'
lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[	 ]\([BCDEGRST][BCDEGRST]*\)[	 ][	 ]*_\([_A-Za-z][_A-Za-z0-9]*\)$/\1 _\2 \2/p'\'' | sed '\''/ __gnu_lto/d'\'''
lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/  {"\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(lib.*\)$/  {"\1", (void *) \&\1},/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/  {"lib\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[BCDEGRST][BCDEGRST]* .* \(.*\)$/extern char \1;/p'\'''
lt_cv_sys_global_symbol_to_import=
lt_cv_sys_max_cmd_len=196608
lt_cv_to_host_file_cmd=func_convert_file_noop
lt_cv_to_tool_file_cmd=func_convert_file_noop
lt_cv_truncate_bin='/bin/dd bs=4096 count=1'
pkg_cv_FONTCONFIG_CFLAGS='-I/usr/local/Cellar/fontconfig/2.13.1/include -I/usr/local/opt/freetype/include/freetype2'
pkg_cv_FONTCONFIG_LIBS='-L/usr/local/Cellar/fontconfig/2.13.1/lib -L/usr/local/opt/freetype/lib -lfontconfig -lfreetype'
pkg_cv_FREETYPE_CFLAGS=-I/usr/local/opt/freetype/include/freetype2
pkg_cv_FREETYPE_LIBS='-L/usr/local/opt/freetype/lib -lfreetype'
pkg_cv_HARFBUZZ_CFLAGS='-I/usr/local/Cellar/glib/2.60.6/include/glib-2.0 -I/usr/local/Cellar/glib/2.60.6/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/harfbuzz/2.5.3/include/harfbuzz -I/usr/local/Cellar/pcre/8.43/include -I/usr/local/Cellar/graphite2/1.3.13/include'
pkg_cv_HARFBUZZ_LIBS='-L/usr/local/Cellar/harfbuzz/2.5.3/lib -lharfbuzz'

## ----------------- ##
## Output variables. ##
## ----------------- ##

ACLOCAL='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/missing aclocal-1.15'
AMDEPBACKSLASH='\'
AMDEP_FALSE='#'
AMDEP_TRUE=''
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
AM_DEFAULT_VERBOSITY='0'
AM_V='$(V)'
AR='ar'
AS='as'
AUTOCONF='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/missing autoconf'
AUTOHEADER='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/missing autoheader'
AUTOMAKE='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/missing automake-1.15'
AWK='gawk'
BUILDOPT_INSTALL_TESTS_FALSE=''
BUILDOPT_INSTALL_TESTS_TRUE=''
CAIRO_CFLAGS=''
CAIRO_LIBS=''
CAIRO_PREFIX=''
CC='/Users/goxberry/repos/spack/lib/spack/env/clang/clang'
CCDEPMODE='depmode=gcc3'
CFLAGS='-g -O2 -Wall'
CPP='/Users/goxberry/repos/spack/lib/spack/env/clang/clang -E'
CPPFLAGS=''
CROSS_COMPILING_FALSE=''
CROSS_COMPILING_TRUE='#'
CXX='/Users/goxberry/repos/spack/lib/spack/env/clang/clang++'
CXXCPP='/Users/goxberry/repos/spack/lib/spack/env/clang/clang++ -E'
CXXDEPMODE='depmode=gcc3'
CXXFLAGS='-g -O2'
CYGPATH_W='echo'
DEFS=''
DEPDIR='.deps'
DLLTOOL='dlltool'
DSYMUTIL='dsymutil'
DUMPBIN=''
ECHO_C='\c'
ECHO_N=''
ECHO_T=''
EGREP='/usr/local/bin/ggrep -E'
ENABLE_DOC_CROSS_REFERENCES_FALSE=''
ENABLE_DOC_CROSS_REFERENCES_TRUE=''
ENABLE_GTK_DOC_FALSE=''
ENABLE_GTK_DOC_TRUE=''
EXEEXT=''
FGREP='/usr/local/bin/ggrep -F'
FONTCONFIG_CFLAGS='-I/usr/local/Cellar/fontconfig/2.13.1/include -I/usr/local/opt/freetype/include/freetype2'
FONTCONFIG_LIBS='-L/usr/local/Cellar/fontconfig/2.13.1/lib -L/usr/local/opt/freetype/lib -lfontconfig -lfreetype'
FREETYPE_CFLAGS='-I/usr/local/opt/freetype/include/freetype2'
FREETYPE_LIBS='-L/usr/local/opt/freetype/lib -lfreetype'
GLIB_CFLAGS=''
GLIB_LIBS=''
GLIB_MKENUMS=''
GLIB_PREFIX=''
GREP='/usr/local/bin/ggrep'
GTKDOC_CHECK='/usr/bin/true'
GTKDOC_CHECK_PATH='/usr/bin/true'
GTKDOC_DEPS_CFLAGS=''
GTKDOC_DEPS_LIBS=''
GTKDOC_MKPDF='/usr/bin/true'
GTKDOC_REBASE='/usr/bin/true'
GTK_DOC_BUILD_HTML_FALSE=''
GTK_DOC_BUILD_HTML_TRUE=''
GTK_DOC_BUILD_PDF_FALSE=''
GTK_DOC_BUILD_PDF_TRUE=''
GTK_DOC_USE_LIBTOOL_FALSE=''
GTK_DOC_USE_LIBTOOL_TRUE=''
GTK_DOC_USE_REBASE_FALSE=''
GTK_DOC_USE_REBASE_TRUE=''
HARFBUZZ_CFLAGS='-I/usr/local/Cellar/glib/2.60.6/include/glib-2.0 -I/usr/local/Cellar/glib/2.60.6/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/harfbuzz/2.5.3/include/harfbuzz -I/usr/local/Cellar/pcre/8.43/include -I/usr/local/Cellar/graphite2/1.3.13/include'
HARFBUZZ_LIBS='-L/usr/local/Cellar/harfbuzz/2.5.3/lib -lharfbuzz'
HAVE_CAIRO_FALSE=''
HAVE_CAIRO_FREETYPE_FALSE=''
HAVE_CAIRO_FREETYPE_TRUE='#'
HAVE_CAIRO_PDF_FALSE=''
HAVE_CAIRO_PDF_TRUE='#'
HAVE_CAIRO_PNG_FALSE=''
HAVE_CAIRO_PNG_TRUE='#'
HAVE_CAIRO_PS_FALSE=''
HAVE_CAIRO_PS_TRUE='#'
HAVE_CAIRO_QUARTZ_FALSE=''
HAVE_CAIRO_QUARTZ_TRUE='#'
HAVE_CAIRO_TRUE='#'
HAVE_CAIRO_WIN32_FALSE=''
HAVE_CAIRO_WIN32_TRUE='#'
HAVE_CAIRO_XLIB_FALSE=''
HAVE_CAIRO_XLIB_TRUE='#'
HAVE_CORE_TEXT_FALSE='#'
HAVE_CORE_TEXT_TRUE=''
HAVE_CXX_FALSE='#'
HAVE_CXX_TRUE=''
HAVE_FREETYPE_FALSE='#'
HAVE_FREETYPE_TRUE=''
HAVE_GTK_DOC_FALSE=''
HAVE_GTK_DOC_TRUE=''
HAVE_HARFBUZZ_FALSE='#'
HAVE_HARFBUZZ_TRUE=''
HAVE_INTROSPECTION_FALSE=''
HAVE_INTROSPECTION_TRUE=''
HAVE_WIN32_FALSE=''
HAVE_WIN32_TRUE='#'
HAVE_XFT_FALSE=''
HAVE_XFT_TRUE='#'
HTML_DIR=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
INTROSPECTION_CFLAGS=''
INTROSPECTION_COMPILER=''
INTROSPECTION_GENERATE=''
INTROSPECTION_GIRDIR=''
INTROSPECTION_LIBS=''
INTROSPECTION_MAKEFILE=''
INTROSPECTION_SCANNER=''
INTROSPECTION_TYPELIBDIR=''
LD='/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld'
LDFLAGS=''
LIBOBJS=''
LIBRARY_LIBTOOL_OPTIONS=''
LIBS=''
LIBTHAI_CFLAGS=''
LIBTHAI_LIBS=''
LIBTOOL='$(SHELL) $(top_builddir)/libtool'
LIB_EXE_MACHINE_FLAG=''
LIPO='lipo'
LN_S='ln -s'
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
MAKEINFO='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/missing makeinfo'
MANIFEST_TOOL=':'
MKDIR_P='/Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/install-sh -c -d'
MSVC_BASE_NO_TOOLSET_SET_FALSE=''
MSVC_BASE_NO_TOOLSET_SET_TRUE=''
MSVC_NO_TOOLSET_SET_FALSE=''
MSVC_NO_TOOLSET_SET_TRUE=''
MS_LIB_AVAILABLE_FALSE=''
MS_LIB_AVAILABLE_TRUE='#'
NM='/usr/bin/nm -B'
NMEDIT='nmedit'
OBJDUMP='objdump'
OBJEXT='o'
OS_WIN32_FALSE=''
OS_WIN32_TRUE='#'
OTOOL64=':'
OTOOL='otool'
PACKAGE='pango'
PACKAGE_BUGREPORT='http://bugzilla.gnome.org/enter_bug.cgi?product=pango'
PACKAGE_NAME='pango'
PACKAGE_STRING='pango 1.41.0'
PACKAGE_TARNAME='pango'
PACKAGE_URL=''
PACKAGE_VERSION='1.41.0'
PANGO_API_VERSION='1.0'
PANGO_BINARY_AGE='4100'
PANGO_CURRENT_MINUS_AGE='0'
PANGO_DEBUG_FLAGS='-DPANGO_ENABLE_DEBUG'
PANGO_HIDDEN_VISIBILITY_CFLAGS=''
PANGO_INTERFACE_AGE='0'
PANGO_VERSION='1.41.0'
PANGO_VERSION_MAJOR='1'
PANGO_VERSION_MICRO='0'
PANGO_VERSION_MINOR='41'
PATH_SEPARATOR=':'
PKGCONFIG_CAIRO_REQUIRES=''
PKG_CONFIG='/usr/local/opt/pkg-config/bin/pkg-config'
PKG_CONFIG_LIBDIR=''
PKG_CONFIG_PATH='/usr/local/opt/cairo/lib/pkgconfig:/usr/local/opt/gobject-introspection/lib/pkgconfig:/usr/local/opt/glib/lib/pkgconfig:/usr/local/opt/harfbuzz/lib/pkgconfig'
PLATFORM_WIN32_FALSE=''
PLATFORM_WIN32_TRUE='#'
RANLIB='ranlib'
SED='/usr/local/bin/gsed'
SET_MAKE=''
SHELL='/bin/sh'
STRIP='strip'
VERSION='1.41.0'
WINDRES=''
XFT_CFLAGS=''
XFT_LIBS=''
ac_ct_AR='ar'
ac_ct_CC='/Users/goxberry/repos/spack/lib/spack/env/clang/clang'
ac_ct_CXX=''
ac_ct_DUMPBIN=''
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE=''
am__fastdepCC_FALSE='#'
am__fastdepCC_TRUE=''
am__fastdepCXX_FALSE='#'
am__fastdepCXX_TRUE=''
am__include='include'
am__isrc=' -I$(srcdir)'
am__leading_dot='.'
am__nodep='_no'
am__quote=''
am__tar='$${TAR-tar} chof - "$$tardir"'
am__untar='$${TAR-tar} xf -'
bindir='${exec_prefix}/bin'
build='x86_64-apple-darwin18.7.0'
build_alias=''
build_cpu='x86_64'
build_os='darwin18.7.0'
build_vendor='apple'
datadir='${datarootdir}'
datarootdir='${prefix}/share'
docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
dvidir='${docdir}'
exec_prefix='NONE'
host='x86_64-apple-darwin18.7.0'
host_alias=''
host_cpu='x86_64'
host_os='darwin18.7.0'
host_vendor='apple'
htmldir='${docdir}'
includedir='${prefix}/include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /Users/goxberry/repos/spack/var/spack/stage/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo/spack-src/install-sh'
libdir='${exec_prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${datarootdir}/man'
mkdir_p='$(MKDIR_P)'
ms_librarian=''
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/Users/goxberry/repos/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.1-apple/pango-1.41.0-evswhej5y6h76dzz7xslifzofhcgjreo'
program_transform_name='s,x,x,'
psdir='${docdir}'
sbindir='${exec_prefix}/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "pango"
#define PACKAGE_TARNAME "pango"
#define PACKAGE_VERSION "1.41.0"
#define PACKAGE_STRING "pango 1.41.0"
#define PACKAGE_BUGREPORT "http://bugzilla.gnome.org/enter_bug.cgi?product=pango"
#define PACKAGE_URL ""
#define PACKAGE "pango"
#define VERSION "1.41.0"
#define PANGO_BINARY_AGE 4100
#define PANGO_INTERFACE_AGE 0
#define PANGO_VERSION_MAJOR 1
#define PANGO_VERSION_MINOR 41
#define PANGO_VERSION_MICRO 0
#define HAVE_SYSCONF 1
#define HAVE_GETPAGESIZE 1
#define STDC_HEADERS 1
#define HAVE_SYS_TYPES_H 1
#define HAVE_SYS_STAT_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define HAVE_MEMORY_H 1
#define HAVE_STRINGS_H 1
#define HAVE_INTTYPES_H 1
#define HAVE_STDINT_H 1
#define HAVE_UNISTD_H 1
#define HAVE_UNISTD_H 1
#define HAVE_SYS_MMAN_H 1
#define HAVE_DLFCN_H 1
#define LT_OBJDIR ".libs/"
#define HAVE_FCWEIGHTFROMOPENTYPEDOUBLE 1
#define HAVE_FREETYPE 1
#define HAVE_CORE_TEXT 1

configure: exit 1
```

### Platform and user environment

I'm running macOS 10.14.6.

and, if relevant, post or attach:

- `packages.yaml`:
```yaml
packages:
  # arpack-ng: # Don't use this version, which links to Accelerate libs
  #   paths:
  #     arpack-ng+mpi@3.6.2 ^openmpi@4.0.1: /usr/local/opt/arpack/3.6.2
  autoconf:
    paths:
      autoconf@2.69 %clang@10.0.1-apple: /usr/local/opt/autoconf
  automake:
    paths:
      automake@1.16.1 %clang@10.0.1-apple: /usr/local/opt/automake
  glib:
    paths:
      glib@2.60.6~libmount %clang@10.0.1-apple: /usr/local/opt/glib
  gobject-introspection: # needs to be updated in spack
    paths:
      gobject-introspection@1.60.2 %clang@10.0.1-apple: /usr/local/opt/gobject-introspection
  harfbuzz:
    paths:
      harfbuzz@2.5.3 %clang@10.0.1-apple: /usr/local/opt/harfbuzz
  libffi:
    paths:
      libffi@3.2.1 %clang@10.0.1-apple: /usr/local/opt/libffi
  libtool:
    paths:
      libtool@2.4.6 %clang@10.0.1-apple: /usr/local/opt/libtool
  m4:
    paths:
      m4@1.4.18 %clang@10.0.1-apple: /usr/local/opt/m4
  pkg-config:
    paths:
      pkg-config@0.29.2 %clang@10.0.1-apple: /usr/local/opt/pkg-config
```

--- a/var/spack/repos/builtin/packages/pango/package.py
+++ b/var/spack/repos/builtin/packages/pango/package.py
@@ -23,7 +23,7 @@ class Pango(AutotoolsPackage):
 
     variant('X', default=False, description="Enable an X toolkit")
 
-    depends_on("pkgconfig", type="build")
+    depends_on("pkgconfig@0.9.0:", type="build")
     depends_on("harfbuzz")
     depends_on("cairo")
     depends_on("cairo~X", when='~X')
@@ -31,6 +31,14 @@ class Pango(AutotoolsPackage):
     depends_on("libxft", when='+X')
     depends_on("glib")
     depends_on('gobject-introspection')
+    depends_on('fontconfig')
+    depends_on('freetype@2:')
+    depends_on('libffi')
+
+    depends_on('harfbuzz@1.2.3:', when='@1.41.0')
+    depends_on('libxft@2.0.0:', when='@1.41.0 +X')
+    depends_on('glib@2.33.12:', when='@1.41.0')
+    depends_on('fontconfig@2.11.91:', when='@1.41.0')
 
     def url_for_version(self, version):
         url = "http://ftp.gnome.org/pub/GNOME/sources/pango/{0}/pango-{1}.tar.xz"

